src/app/shared/shared.module.ts

```diff
@@ -42,6 +43,7 @@ export class SharedModule {
                 CoreWebService,
                 DotEventsService,
                 DotNavigationService,
+                DotDialogMessageService,
```

Comment:
Instead of doing this, we can take advantage of the provideIn property in the injectables